### PR TITLE
cpp changelog for Jun Release

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: releases/2021-06/cpp.md [hunk 1]
```diff
@@ -10,22 +10,29 @@ The Azure SDK team is pleased to make available the June 2021 client library rel
...
+- Azure Storage Common
+- Azure Storage Blobs
+- Azure Storage Files Shares
+- Azure Storage Files DataLake
 
 #### Beta
 
-- _Add packages_
+- Azure Key Vault Common
+- Azure Key Vault
```
Comment (antkmsft) [inline releases/2021-06/cpp.md:23]: Is it Key Vault Keys, or just Key Vault?
And the Security, is it the part of the name, or not?

File: releases/2021-06/cpp.md [hunk 2]
```diff
@@ -10,22 +10,29 @@ The Azure SDK team is pleased to make available the June 2021 client library rel
...
-
-#### Updates
-
-- _Add packages_
+- Azure Core
+- Azure Identity
+- Azure Storage Common
+- Azure Storage Blobs
+- Azure Storage Files Shares
+- Azure Storage Files DataLake
```
Comment (antkmsft) [inline releases/2021-06/cpp.md:18]: If we split into words, probably should be "Data Lake" with the space. 
The brand name has space.

**Review by antkmsft — APPROVED** [review]

File: releases/2021-06/cpp.md (hunk 2 above)
Comment (vhvb1989) [reply releases/2021-06/cpp.md:18]: We have been using `DataLake`, but I would follow @Jinming-Hu / @vinjiang

File: releases/2021-06/cpp.md (hunk 1 above)
Comment (vhvb1989) [inline releases/2021-06/cpp.md:23]:
```suggestion
- Azure Key Vault Keys
```

File: releases/2021-06/cpp.md [hunk 3]
```diff
@@ -10,22 +10,29 @@ The Azure SDK team is pleased to make available the June 2021 client library rel
...
 To install any of our packages, copy and paste the following commands into a terminal:
 
 ```bash
-$> 
+
+# From Source
+git clone https://github.com/Azure/azure-sdk-for-cpp
+# git checkout <tag_name>
+# For example:
+git checkout azure-security-keyvault-keys_4.0.0-beta.1
```
Comment (vhvb1989) [inline releases/2021-06/cpp.md:35]:
```suggestion
git checkout azure-storage-blobs_12.0.0
```

File: releases/2021-06/cpp.md [hunk 4]
```diff
@@ -34,9 +41,72 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+- Removed `IfUnmodifiedSince` from access conditions of setting filesystem metadata operation.
+- Updated some samples.
+- Fixed a read consistency issue.
+
+### azure-security-keyvault-common [Changelog](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/keyvault/azure-security-keyvault-common/CHANGELOG.md#400-beta3-2021-06-08)
+
+No breaking changes or new features added. Includes only implementation enhancements.
+
+
```
Comment (vhvb1989) [inline releases/2021-06/cpp.md:98]:
```suggestion

```

File: releases/2021-06/index.md [hunk 5]
```diff
@@ -23,7 +23,6 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 * [JavaScript/TypeScript release notes](js.md)
 * [Python release notes](python.md)
 * [C++ release notes](cpp.md)
-* [Embedded C release notes](c.md)
```
Comment (vhvb1989) [inline releases/2021-06/index.md:26]: @ahsonkhan confirm we didn't release C on June right?

Comment (ahsonkhan) [reply releases/2021-06/index.md:26]: That is correct, yes :)

**Comment by weshaggard** [comment]
@vhvb1989 please have a look at PR https://github.com/Azure/azure-sdk/pull/2995. It seems like we are in a good place to start having our automation pull in the release notes for cpp. If you want to fully onboard then we will need to update the release template for cpp so it can start picking up the data from the cpp.yml file we can generate from the changelogs.

**Comment by vhvb1989** [comment]
> @vhvb1989 please have a look at PR #2995. It seems like we are in a good place to start having our automation pull in the release notes for cpp. If you want to fully onboard then we will need to update the release template for cpp so it can start picking up the data from the cpp.yml file we can generate from the changelogs.

Thank you Wes!  I have approved the PR. Looking forward to see it working

**Comment by weshaggard** [comment]
@vhvb1989 please work with myself or @chidozieononiwu to get your cpp template file updated to start consuming the data from the yml file.

**Pull request merged**